ph5/clients/tests/test_ph5tostationxml.py

```diff
@@ -331,23 +445,41 @@ def setUp(self):
                                  'ph5/test_data/segd/3ch.fcnt')]
         with patch.object(sys, 'argv', testargs):
             segd2ph5.main()
-        self.ph5sxml, self.mng, self.parser = getParser(
-            self.tmpdir, "master.ph5", "CHANNEL")
 
     def tearDown(self):
         self.mng.ph5.close()
-        super(TestPH5toStationXMLParser_resp_load_not_run, self).tearDown()
+        super(TestPH5toStationXMLParser_gen_resp_issue, self).tearDown()
 
-    def test_create_obs_network(self):
-        self.parser.manager.ph5.read_experiment_t()
-        self.parser.experiment_t = self.parser.manager.ph5.Experiment_t['rows']
-        self.parser.add_ph5_stationids()
-        with LogCapture() as log:
-            log.setLevel(logging.ERROR)
-            self.parser.create_obs_network()
-            self.assertEqual(log.records[0].msg,
-                             'All response file names are blank in response '
-                             'table. Check if resp_load has been run.')
+    def test_read_networks_no_resp_filename(self):
+        self.ph5sxml, self.mng, self.parser = getParser(
+            self.tmpdir, "master.ph5", "CHANNEL")
+        with self.assertRaises(ph5tostationxml.PH5toStationXMLError) as contxt:
+            self.parser.read_networks()
+        self.assertEqual(
+            contxt.exception.message,
+            "Response table does not contain any response file names. "
+            "Check if resp_load has been run or if metadatatoph5 input "
+            "contained response information.")
+
+    def test_read_networks_response_duplication(self):
```

Comment:
This is the test that I was trying to create, I previously missed it, and I will delete the test instances that I pushed last night. 
The problem is that the master branch will output a stationxml files from a ph5 experiment with duplicated response n_i values in the responses table. It seems like this behavior should likely be mimicked if the flag --stationxml_on_error=True is included in the ph5tostationxml call. I previously suggested that we use a  FIFO logical approach and am open to other ideas. What do you think?